pom.xml

```diff
@@ -269,7 +269,19 @@
         <dependency>
             <groupId>uk.gov.pay</groupId>
             <artifactId>testing</artifactId>
-            <version>1.0.0-7</version>
+            <version>1.0.0-95d787ff5931ee0d5e54afd7fbc3834cf772dad4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>1.1.4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Not certain but I think this might already be covered by the GlassFish dependency